### DISCUSSION TITLE
Error 500 when loading dashboard

### DISCUSSION THREAD

**Discussion by stienjo1993** [KelvinTegelaar/CIPP#1538] [Q&A]
Hi All,

With the latest version installed 3.5.1 I got the following error code:
A generic error has occurred.
500 - Request failed with status code 500
When I select a tenant this error will not occure on the Dashboard, when I select all tenants this error occures. 
Someone having this same problem? maybe a bug in the latest version?

**Comment by KelvinTegelaar** [comment]
We don't use this discussion board for potential support issues etc. If you want support you can start a sponsorship.
